### PR TITLE
SYS-1531: Make square thumbnails

### PR DESCRIPTION
**Pull request by ztucker4**
Implements [SYS-1531](https://uclalibrary.atlassian.net/browse/SYS-1531)

Currently, the OH staff application creates both submaster and thumbnail derivative images in the same way: resize the image such that the longest dimension is the maximum size for the derivative type (750 px for submasters, 200 px for thumbnails), maintaining the aspect ratio. 

This PR changes the way that thumbnails are generated. Now, thumbnails are always 200x200 squares, with black bars vertically or horizontally bordering the resized image to fill any extra space. Example images are included on the [Jira ticket](https://uclalibrary.atlassian.net/browse/SYS-1531).

This is a small code change done entirely within the `ImageFileHandler` class. An additional test image (public domain, from Wikimedia Commons) is included to demonstrate the case that a master image is taller than it is wide. The other case (wider image) is demonstrated by the existing `sample_marbles.tif` file.

[SYS-1531]: https://uclalibrary.atlassian.net/browse/SYS-1531?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: oh_staff_ui/classes/ImageFileHandler.py [hunk 1]
```diff
@@ -127,3 +129,17 @@ def _get_new_image_dimensions(
         scale_factor = max(current_sizes) / max_size
         new_sizes = tuple(int(dimension / scale_factor) for dimension in current_sizes)
         return new_sizes
+
+    def _make_square_thumbnail(self, thumbnail_image: Image.Image) -> Image.Image:
```
Comment (akohler) [inline oh_staff_ui/classes/ImageFileHandler.py:133]: Please add a test for the thumbnail size to `MediaFileTestCase`, to confirm the size is 200x200 (more appropriately, `settings.IMAGE_SETTINGS["thumbnail_long_dimension"]` square).

File: oh_staff_ui/classes/ImageFileHandler.py [hunk 2]
```diff
@@ -127,3 +129,17 @@ def _get_new_image_dimensions(
...
+        """Create a square thumbnail from a rectangular image, with black padding.
+        To be used after image is resized to thumbnail size.
+
+        Arguments:
+        thumbnail_image: image resized such that the longest side is the thumbnail size.
+        Returns: the square thumbnail.
+        """
+        x, y = thumbnail_image.size
```
Comment (akohler) [inline oh_staff_ui/classes/ImageFileHandler.py:141]: Please replace `x` and `y` with `width` and `height`.  It doesn't matter for the math and we can infer it from documentation of `_get_new_image_dimensions()`, but making it explicit in the code means we don't have to wonder about it.

**Review by akohler — CHANGES_REQUESTED** [review]
@ztucker4 I processed a couple of images locally and confirmed that they match the examples in the Jira ticket.  Just a couple of minor requested changes to make the code clearer and to test for thumbnail size.

Please also add to release notes and bump the version for release, then I'll merge.  Thanks.

**Comment by ztucker4** [comment]
This is ready for re-review - I've fixed the variable names, added a new test for thumbnail size (`test_thumbnail_image_size`), and updated the tag version and release notes.

**Review by akohler — APPROVED** [review]

**Pull request merged**